omnibus/files/private-chef-ctl-commands/chef-server-ctl

```diff
@@ -19,6 +18,11 @@ module Omnibus
       d
     end
 
+    def credentials
```

Comment:
Not required in this PR, but we should clean up commands we already fixed to use this function rather than instantiating Veil objects.